### DISCUSSION TITLE
Using FMI.jl with dSPACE hardware

### DISCUSSION THREAD

**Discussion by YJR78** [ThummeTo/FMI.jl#262] [General]
Hello,
I m currently using dSPACE hardware and that's the last task that requires matlab simulink and related toolboxes in my workflow ! All other tasks has been switched to Julia! Do you think that it would be possible to integrate the FMI version 3 "SE" model export ? as I think that those model could be implemented in dSPACE hardware using configuration desk.

**Comment by ThummeTo** [comment]
Hi,
thanks for the request!

Just that I got this right: Do you want to _export_ a FMU from Julia (as FMI3-SE) or do you want to _import_ a FMU (FMI3-SE) into Julia?
That's a difference regarding the workload :-)

For now, we don't use FMI3-SE and not many tools support it that's why it's low-prio on our road map (but part of it of course to have the complete standard implemented).
However, we are happy to _support_ development by guiding/reviewing or similar.

Best!

**Reply by YJR78** [reply]
Hello,
I would like to export FMI3-SE from Julia ... i.e. designing controller with control tool box implement it as a FMI3-SE and use it in RCP hardware like microlabbox from dSPACE. Best regards

**Reply by ThummeTo** [reply]
Ah I see, sounds cool!

We are currently on restructuring the FMIExport, as soon as ME is working properly a CS and SE-implementation should be doable with reasonable effort. (ME is the most work and starting point for CS/SE)
